Where_Is_My_Wife/Assets/_Main/Scripts/SceneManagement/FreeplayButton.cs

```diff
@@ -26,6 +27,12 @@ protected override void ChangeScene()
             LevelManager.Instance.SetStoryMode(false);
             base.ChangeScene();
         }
+
+        public override void OnSelect(BaseEventData eventData)
+        {
+            base.OnSelect(eventData);
+            MenuManager.Instance.ChangeBackground(_levelNumber - 1);
```

Comment:
The `-1` should be inside the function, not here.

Reply:
This one is necessary here because _levelNumber details the level after it to be unlocked, and I need ChangeBackground to stay the same, otherwise I need to change ChangeToLastPlayedStoryModeLevelBackground().